CHANGELOG.md

```diff
@@ -13,6 +13,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Changes to make MAPL 2 History output match GEOS FP file spec
```

Comment:
@mathomp4 - I wonder if this should be in the ### Fixed section instead?

Reply:
I think changed is appropriate. I mean, nothing was "broken" in that it was perfectly fine code.

Reply:
I won't debate the issue for the sake of urgency.  But "fixed" is for anything where the result is not as intended.   Relatively harmless in this case, but still incorrect from an ops consistency point-of-view.   "changed" is more for intentional evolution of features.  But as long as it is in the changelog the philosophers can debate.